File: src/main/java/guru/xgm/language/vjass/psi/VjassTokenSets.java
package guru.xgm.language.vjass.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.language.vjass.psi.VjassTypes.BLOCK_COMMENT;
import static guru.xgm.language.vjass.psi.VjassTypes.LINE_COMMENT;

public interface VjassTokenSets {
    TokenSet COMMENTS = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT);
}

File: src/main/kotlin/guru/xgm/language/vjass/psi/VjassTokenSets.kt
package guru.xgm.language.vjass.psi

import com.intellij.psi.tree.TokenSet

interface VjassTokenSets {
    companion object {
        @JvmField
        val COMMENTS: TokenSet = TokenSet.create(VjassTypes.LINE_COMMENT, VjassTypes.BLOCK_COMMENT)
    }
}
